tokenManager.js

```diff
@@ -0,0 +1,97 @@
+require("dotenv").config();
```

Comment:
`require()` isn't available for JavaScript modules -- this will need to be converted to an `import` statement.